src/Tree/style.css

```diff
@@ -3,14 +3,14 @@
   height: 100%;
 }
 
-.rd3t-grabbable {
-  cursor: move; /* fallback if grab cursor is unsupported */
-  cursor: grab;
-  cursor: -moz-grab;
-  cursor: -webkit-grab;
-}
-.rd3t-grabbable:active {
-    cursor: grabbing;
-    cursor: -moz-grabbing;
-    cursor: -webkit-grabbing;
-}
+/*.rd3t-grabbable {*/
```

Comment:
Can you explain why this is necessary? 

I don't know if it's a breaking change strictly speaking, but this would change cursor behaviour in a significant way.

Reply:
HI,
no, it's not necessary, I commented in order to ease the link mouseout functionality.